doc/api/magic-snippets.md

```diff
@@ -104,10 +121,9 @@ Trying to delete the `chinstrap` snippet will display an error message:
 %sqlcmd snippets -d chinstrap
 ```
 
-If you still wish to delete this snippet, you can run the below command:
+If you still wish to delete this snippet, you should use `force-delete` by running the below command:
```

Comment:
Have u checked these sentences using Grammarly?

Reply:
<img width="800" alt="Screen Shot 2023-06-29 at 8 39 21 AM" src="https://github.com/ploomber/jupysql/assets/43221691/69699f0f-2075-4757-9b93-ba17820824c5">

This sentence seems like there is no issue and currently checking other sentences with Grammarly. Thanks for pointing out this and other comments.

Reply:
No need to add screenshot here. Just make sure the sentences are verified through Grammarly